name_test.go

```diff
@@ -49,24 +47,29 @@ func TestName_Split(t *testing.T) {
 }
 
 func TestJSON(t *testing.T) {
-	type JWT struct {
```

Comment:
The JWT references here seemed like an indirection that made the intent of the test slightly harder to understand so I renamed things to be slightly more generic.